internal/config/config.go

```diff
@@ -119,12 +121,16 @@ func setNonCliDefaultValues(v *viper.Viper) {
 	v.SetDefault("kubeconfig.anchore.account", "admin")
 	v.SetDefault("anchore.http.insecure", false)
 	v.SetDefault("anchore.http.timeout-seconds", 10)
+	v.SetDefault("kubernetes-request-timeout-seconds", -1)
```

Comment:
This is a duplicate I think. It will end up being 60.

Reply:
There's only a 1 character difference, that's the backwards compatible config

`kubernetes-request-timeout-seconds` vs `kubernetes[.]request-timeout-seconds`

Reply:
My bad, misread the string :)